assets/src/edit-story/components/panels/videoPoster.js

```diff
@@ -40,6 +40,7 @@ const Button = styled.button`
 const Img = styled.img`
   width: 100%;
   max-height: 300px;
+  object-fit: contain;
 `;
 
 function VideoPosterPanel({ selectedElements, onSetProperties }) {
```

Comment:
Adding unit tests for this component I figured issues related to WordPress. It is reporting `ReferenceError: wp is not defined`, its a problem related to the `useMediaPicker` hook which is using `wp.media` directly and Jest can't recoginize this, so I would like to get additional insights or suggestions from the team. Do you already figure out a problem like this? cc @barklund @ndev1991 @swissspidy

Reply:
You should be able to mock anything under the `wp` global by defining it in `tests/js/setup-globals.js`, i.e. `global.wp = jest.fn()`. In the old plugin we used something like this for similar situations:

```js
// Set up `wp.*` aliases.
global.wp = {
	media: {
		controller: {
			Library: jest.fn(),
			Cropper: {
				extend: jest.fn(),
			},
		},
		View: {
			extend: jest.fn(),
		},
		view: {
			Toolbar: {
				Select: {
					extend: jest.fn(),
				},
			},
			MediaFrame: {
				Select: {
					extend: jest.fn(),
				},
			},
		},
	},
};
```

You could also mock globals just in the specific test file you are writing instead of globally this way.

Reply:
Nice! Thank you for the suggestion @swissspidy. I will take a look on this approach 👍

Reply:
We should also look into how we can abstract this usage better to simplify storybook integration.

cc @spacedmonkey